src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -851,10 +857,13 @@ public void onHandleTransactionMessage(PeerConnection peer, TransactionMessage t
         broadcast(trxMsg);
       }
     } catch (TraitorPeerException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either log or rethrow this exception. ([squid:S1166](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0plHK86Hco_1tP&open=AWalU-0plHK86Hco_1tP)

[ISSUE_KEY]: AWalU-0plHK86Hco_1tP